pkg/tekton/pipelines.go

```diff
@@ -27,13 +27,17 @@ func GeneratePipelineActivity(buildNumber string, branch string, gitInfo *gits.G
 	name := gitInfo.Organisation + "-" + gitInfo.Name + "-" + branch + "-" + buildNumber
 	pipeline := gitInfo.Organisation + "/" + gitInfo.Name + "/" + branch
 	log.Infof("PipelineActivity for %s", name)
+	var toMerge map[string]string
```

Comment:
@abayer heads up, @dgozalo's latest changes related to batch builds changed the way you can use `step create task`.

Without this change here you cannot just call `jx step crate task` anymore. You also need to specify `--clone-git-url <repo>` and `-e PULL_REFS=<valid-pull-ref>`.

This change will get the old behaviour back.

Reply:
Nice, good catch.